internal/action/commands.go

```diff
@@ -981,7 +958,7 @@ func (s *Action) GetCommands() []*cli.Command {
 	for _, be := range backend.CryptoRegistry.Backends() {
 		bc, ok := be.(commander)
 		if !ok {
-			debug.Log("Backend %s does not implement commander interface\n", be)
+			// debug.Log("Backend %s does not implement commander interface\n", be)
```

Comment:
Why remove this debug log? Feels like it could be useful sometimes, no?

Reply:
These were left overs. We're already way to verbose in the debug log. I'd rather remove those.